tests/e2e/vsan_stretched_cluster_utils.go

```diff
@@ -927,17 +933,28 @@ func triggerFullSyncInParallel(ctx context.Context, client clientset.Interface,
 	if err != nil {
 		framework.Logf("Full sync did not finish in given time, ignoring this error: %v", err)
 	}
-
-	crd := getTriggerFullSyncCrd(ctx, client, cnsOperatorClient)
+	crd, err := getTriggerFullSyncCrd(ctx, client, cnsOperatorClient)
+	if err != nil {
+		framework.Logf("couldn't get full sync crd in given time, ignoring this error: %v", err)
+	}
 	framework.Logf("INFO: full sync crd details: %v", crd)
 	updateTriggerFullSyncCrd(ctx, cnsOperatorClient, *crd)
```

Comment:
what I'm telling is
if you get errors before first fullsync is triggered fail the test
post which for any error, log the error and exit from the subroutine, no need to run the rest of the things